### PR TITLE
Added chart type parameter to constructor

### PR DESCRIPTION
**Pull request by tomwayson**
See comments on https://github.com/Esri/cedar/commit/daf559cff1a97232ac80ee2dde6785aa4221cc2e for details.

### CODE REVIEW

**Comment by tomwayson** [comment]
Note that while updating the example pages to use the new `type` parameter, I also update the code snippets (markdown) to match the script tag on the page (by copying and pasting the content of the script tag into the markdown) since many were out of date.

File: src/cedar.js [hunk 1]
```diff
@@ -578,6 +628,8 @@ Cedar.getJson = function( url, callback ){
 
 
 Cedar._mixin = function(source) {
+    /*jshint loopfunc: true*/
```
Comment (tomwayson) [inline src/cedar.js]: Addeed comment to allow cedar.js to pass jshint

**Comment by ajturner** [comment]
woot! :clapper:

**Comment by abakirov** [comment]
@ajturner 
Cool, thanks! Do you think I should go ahead and update to latest Cedar code or do you expect major API changes?

**Comment by ajturner** [comment]
There will be one more round of word changes but I don't expect major shifts. Working on the labeling which will mean exposing some options to turn on/off auto labels and forced placement

**Pull request merged**